js/pagination.jsx

```diff
@@ -0,0 +1,143 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2014, Trevor Landau <>
```

Comment:
Oh, for npm-installable dependency, we might need to use a `package.json` instead of putting it directly into the repo. But you don't need to do this for this PR. Let't create a followup bug for that.